content/tutorials/getusermedia/intro/en/index.html

```diff
@@ -344,43 +341,40 @@ <h3 id="toc-basic-demo">Basic demo</h3>
   <p><button class="capture-button">Capture video</button> <button id="stop-button">Stop</button></p>
 </div>
 
-<h2 id="toc-screenshot">Taking screenshots</h2>
+<h2 id="toc-screenshot">Take screenshots</h2>
 
 <p>The <code>&lt;canvas&gt;</code> API's <code>ctx.drawImage(video, 0, 0)</code> method makes it trivial to draw
-<code>&lt;video&gt;</code> frames to <code>&lt;canvas&gt;</code>. Of course, now that we have video
-input via <code>getUserMedia()</code>, it's just as easy to create a photo booth application
-with realtime video:</p>
-<pre class="prettyprint"><code>&lt;video autoplay&gt;&lt;/video&gt;
-&lt;img src=""&gt;
-&lt;canvas style="display:none;"&gt;&lt;/canvas&gt;
-
-&lt;script&gt;
-const captureVideoButton =
-  document.querySelector('#screenshot .capture-button');
-const screenshotButton = document.querySelector('#screenshot-button');
-const img = document.querySelector('#screenshot img');
-const video = document.querySelector('#screenshot video');
-
-const canvas = document.createElement('canvas');
-
-captureVideoButton.onclick = function() {
-  navigator.mediaDevices.getUserMedia(constraints).
-    then(handleSuccess).catch(handleError);
+<code>&lt;video&gt;</code> frames to <code>&lt;canvas&gt;</code>. Of course, now that you have video
+input through <code>getUserMedia()</code>, it's just as easy to create a photo-booth app
+with real-time video:</p>
+<pre class="prettyprint"><code>const captureVideoButton = document.querySelector(
```

Comment:
Please re-add `<video autoplay...`

Reply:
Can you provide the full code as it should appear so that I can see where to place this?

Reply:
Done.